OIPA/iati/parser/IATI_2_03.py

```diff
@@ -4882,6 +4883,69 @@ def iati_activities__iati_activity__tag__narrative(self, element):
         activity_tag = self.get_model('ActivityTag')
         self.add_narrative(element, activity_tag)
 
+    def iati_activities__iati_activity__fss(self, element):
+        """"(optional) <fss> element inside <iati_activities/iati-activity>
+        element in 2.03
+        """
+        # we need to check if this element occurs more than once in  the
+        # parent element, which is <iati-activity>
+
+        # FIXME: should database relation be changed to OnetoOne?? see #964
+        activity = self.get_model('Activity')
+        if 'Fss' in self.model_store:
```

Comment:
Correct me if I'm wrong, but if this element can only occur once, I think just checking if `'Fss'` is in model store would be enough.

But I think there's a simpler way how to check this. You could just get check if `Activity` that is in model store has any `fss` objects assigned to it. Something like that:

```
if activity.fss_set.all():
    raise Exception('this element can only occur once')
```

^ because if `<fss>` element would appear twice in a row, I presume `Activity` that's in the memory would already have a `fss` element tied to it (because this particular parser method would be ran twice)

Also finally, I can see that there's no `<fss>` element in the "example_203_file" ([it's here](https://github.com/zimmerman-zimmerman/OIPA/blob/develop/OIPA/static/xml/example_203_file.xml)), so I personally would find it very difficult to implement these things without actual parsing and seeing (for example) what exactly happens when 2 `fss` elements in a row appear. I suggest you should update this example XML file with this new element.